CHANGELOG.md

```diff
@@ -28,6 +28,11 @@ FEATURES:
    environment variable. See the [environment variable
    documentation](https://www.vaultproject.io/docs/commands/environment.html)
    for more details. [GH-1594]
+ * **Service Discovery in `vault init`**: `-auto` option on `vault init`
+   will perform service discovery using Consul. When only one node is
+   discovered, an initialization attempt will be made and when more than
```

Comment:
Rather than "an initialization attempt" in this blocks, since it's high-level, just say "it will be initialized".

"...discovered, it will be initialized. When more than one node is found, their addresses will be output for easy selection."